store/postgres_graphql.go

```diff
@@ -289,6 +301,11 @@ func psqlSubQuery(tenant string, graph *SchemaGraph, qb *sqlQueryBuilder, root *
 		}
 	}
 
+	// By default if no order_by specified, order by the primary key in DESC order
+	if orderByArg == nil && qb.depth == 1 {
+		qb.sql = qb.sql.OrderBy(tableColumn(tc.alias, tableIDField) + " DESC")
```

Comment:
There are string constants defined for the sorting order: `orderAsc` and `orderDesc` at the top of this file :)